fdb-record-layer-core/src/main/proto/record_query_plan.proto

```diff
@@ -91,9 +91,10 @@ message PType {
       optional int32 field_index = 3;
     }
 
-    optional string name = 1;  // referential name -- not used in the planner
-    optional bool is_nullable = 2;
-    repeated PField fields = 3;
+    optional int32 reference_id = 1;
+    optional string name = 2;  // referential name -- not used in the planner
+    optional bool is_nullable = 3;
+    repeated PField fields = 4;
```

Comment:
I suppose it is ok that the field numbers are redefined considering that plan serialization is still under development, right? in other words, we will not deserialize older messages with the (now inconsistent) field numbers.

Reply:
That's right!